files/en-us/glossary/transferable_objects/index.html

```diff
@@ -0,0 +1,26 @@
+---
+title: Transferable Objects
+slug: Glossary/Transferable_Objects
```

Comment:
I think MDN style is this:

```suggestion
title: Transferable objects
slug: Glossary/Transferable_objects
```